index.html

```diff
@@ -1,10 +1,11 @@
 <html>
     <head>
         <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
-        <script src="string_score.min.js"></script>
+        <script src="./resources/scripts/string_score.min.js"></script>
```

Comment:
Change all the `./resources/` in this file to `resources/`. And we're good to go. :+1: